docs/dashboards/subscriptions.md

```diff
@@ -80,7 +80,7 @@ If you check this box, Metabase will drop any visualization settings applied to
 
 Here you can specify which questions Metabase should attach results for.
 
-The attached files will include up to 2000 rows by default. If you're self-hosting Metabase, you can adjust this row limit by setting the environment variable [MB_UNAGGREGATED_QUERY_ROW_LIMIT](../configuring-metabase/environment-variables.md#mb_unaggregated_query_row_limit).
+The attached files will include up to 2000 rows by default. If you're self-hosting Metabase, you can adjust this row limit by setting the environment variable [MB_UNAGGREGATED_QUERY_ROW_LIMIT](../configuring-metabase/environment-variables.md#mb_unaggregated_query_row_limit). If you are using Metabase Cloud, you can [contact Support](https://www.metabase.com/help/premium) and request this limit to be adjusted for your Metabase.
```

Comment:
It's a bit redundant but I still want to leave this here explicitly because this is by far the most common env var adjustment request